### DISCUSSION TITLE
PostgREST Output in XML

### DISCUSSION THREAD

**Discussion by mchatrvd** [PostgREST/postgrest#3163] [Q&A] [answered]
Hi 
I am using the default curl command - curl http://localhost:<port number>/<database object> -X GET  -H "Content-Type: application/json"

This gives me output in the json format. How can i get the output in xml format? I tried using - curl http://localhost:<port number>/<database object> -X GET  -H "Content-Type: application/xml" but its still giving me json output.

Thanks in advance.

**Comment by laurenceisla** [comment, marked as answer]
Hey!
`application/xml` is not a [builtin Media Type in PostgREST](https://postgrest.org/en/latest/references/api/resource_representation.html#builtin-media-type-handlers), so it won't return the data in `xml` format by default. You'll need to define it yourself by using [Media Type Handlers](https://postgrest.org/en/latest/references/api/media_type_handlers.html) and `GET` the data with the `Accept: application/xml` header (there's an example in that page of the docs).
